Cargo.toml

```diff
@@ -11,7 +11,7 @@ license = "Apache-2.0 OR MIT"
 [dependencies]
 libc = ">=0.2.39"
 kvm-bindings = { version = ">=0.2.0", features = ["fam-wrappers"] }
-vmm-sys-util = ">=0.1.1"
+vmm-sys-util = ">=0.2.1"
```

Comment:
Should it be 0.2.0?

Reply:
0.2.1 contains a fix for clone of FamSafeWrappers which were released in 0.2.0, I chose to enforce using the fixed version of the crate.